app/Helpers/ID_hasher.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Helpers;
+
+use Vinkla\Hashids\Facades\Hashids;
+
+class ID_hasher
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class "ID_hasher" to match the regular expression ^[A-Z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS101)

Reply:
The name `ID_hasher` is not [PSR1 compliant](http://www.php-fig.org/psr/psr-1/).

Please change it everywhere with `idHasher`.